pwned/validators.py

```diff
@@ -15,5 +20,10 @@ class PwnedValidator:
     def validate(self, password, user=None):
         pwned_client = self.client()
         count = pwned_client.count_occurrences(password)
+
         if count >= get_config()['OCCURRENCE_THRESHOLD']:
-            raise ValidationError(self.message, code=self.code)
+            if get_config()['LOG_ONLY']:
+                logger.warn('Password used with %d occurrences in Pwned Password', count, extra={'user': user})
```

Comment:
Not sure that the log level should change depending on whether you have set `LOG_ONLY`? I think it should always be logged at the same level (warn), and the setting should only control whether a validation error is raised.

Reply:
It doesn't make sense to me to log a warning when we catch a validation failure? The only reason I log it at all is to allow projects to monitor how many people are being hit with the validation failure.